examples/mobile/HomeApp/webclip/weather/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "weather",
+    "fullname": "Weather",
```

Comment:
how about camelCase notation?
fullName, cardType etc..

Reply:
That's a good idea, camel case is natural programming notation in JS objects, I will do it